pkgs/tools/misc/ugtrain/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, autoconf, automake, binutils, glib, libtool, pkg-config, scanmem, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.1";
+  pname = "ugtrain";
+
+  src = fetchFromGitHub {
+    owner  = "ugtrain";
+    repo   = "ugtrain";
+    rev    = "v${version}";
+    sha256 = "0pw9lm8y83mda7x39874ax2147818h1wcibi83pd2x4rp1hjbkkn";
+  };
+
+  nativeBuildInputs = [ autoconf automake libtool scanmem ];
+  buildInputs = [ binutils pkg-config ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/ugtrain/ugtrain";
+    description = "The Universal Elite Game Trainer for CLI (Linux game trainer research project)";
+    maintainers = with maintainers; [ mtrsk ];
+    platforms = platforms.linux;
+    license = licenses.gpl3;
```

Comment:
gpl3 is deprecated, consider using gpl3Only or gpl3Plus (https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206/3).